.github/workflows/builds.yml

```diff
@@ -299,8 +299,8 @@ jobs:
           CSC_IDENTITY_AUTO_DISCOVERY: false
           SNAPCRAFT_LOGIN: ${{ secrets.SNAPCRAFT_LOGIN }}
```

Comment:
I would simply change `SNAPCRAFT_LOGIN` by `SNAPCRAFT_STORE_CREDENTIALS` as this will store the value of the secret into the expected environment variable.